repos/fdbt-site/tests/pages/globalSettingsCapValidity.test.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import CapValidity, { getFieldset } from '../../src/pages/selectCapValidity';
+import {
+    mockSelectCapValidityFieldset,
+    mockSelectCapValidityFieldsetWithErrors,
+    mockSelectCapValidityFieldsetWithInputErrors,
+} from '../testData/mockData';
+import { ErrorInfo } from '../../src/interfaces';
+
+describe('pages', () => {
+    describe('capValidity', () => {
```

Comment:
Why is this file called globalSettingsCapValidity?

Reply:
This file, and the globalSettingsPeriodValidity - neither of them should say globalSettings in the name. Please could you remove it from both? The snapshots probably need remaking (npm run test will explain).